src/script/view_model/panel/GuestsAndServicesViewModel.js

```diff
@@ -20,6 +20,7 @@
 import Logger from 'utils/Logger';
 
 import BasePanelViewModel from './BasePanelViewModel';
+import ClipboardUtil from '../../util/ClipboardUtil';
```

Comment:
There is an alias to access utils classes/functions.

```suggestion
import ClipboardUtil from 'utils/ClipboardUtil';
```